README.md

```diff
@@ -39,6 +39,54 @@ If you're looking to utilize the capabilities of Starknet contracts within the R
 
 3. **Feedback**: Your feedback is invaluable to us 🌟! If you encounter any issues or have game-changing suggestions, don't hesitate to reach out through our [Discord](https://discord.com/invite/PaCMRFdvWT) or our [Community Forum](https://community.nethermind.io/). Let's make something awesome together! 🤝
 
+### Troubleshooting
+
+Most issues with Starknet plugin or Remix itself are caused by connectivity problems (also resulting from restricted networks, web-proxies blocking certain content, etc.) or browser plugin interference. 
+- A user should first attempt to disable any browser components & addons which may impact the connectivity or Javascript execution. 
+- Some networks may restrict connectivity to certain sites or domains. Using a VPN connection may resolve problems observed on restricted networks.
+
+More specific potential error causes are also described in detail below. 
+
+<details>
+<summary><strong>1. Problems when searching plugins in Remix UI</strong></summary>  
+<br/>
+Searching for online plugins in Remix IDE may return blank or otherwise incorrect content, such as on screenshot below:
+
+![Plugin search not working](docs/images/plugin-search-error.png)
+Ensure that the following URL is accessible from a web browser, and that a JSON metadata code is returned -  
+https://raw.githubusercontent.com/ethereum/remix-plugins-directory/master/build/metadata.json :
+
+![Correct plugin metadata](docs/images/plugin-metadata-connectivity.png)
+
+</details>
+
+<details>
+<summary><strong>2. Problems when launching Starknet plugin in Remix UI</strong></summary> 
+<br/>  
+Starknet plugin launch issues may be caused by connectivity errors or plugin components being unavailable. This can be tested from web browser, as indicated below:
+
+- https://cairo-remix-test.nethermind.io should respond with blank page (advanced: viewing page source will reveal a React component entry HTML markup) - no errors should be reported by the browser
+- https://cairo-compile-remix-test.nethermind.io/health should respond with `OK`
+- https://starknet-devnet-dev.nethermind.io/feeder_gateway/get_transaction_status?transactionHash=0xbadfad should respond with `{"tx_status":"NOT_RECEIVED"}`
```

Comment:
maybe here add an example from the rpc endpoint also, since feeder gateway will be deprecated soon (in few months),

Reply:
Actually I've replaced it with generic https://starknet-devnet-dev.nethermind.io/predeployed_accounts - whish is just enough to check devnet connectivity.